contracts/upgradeable_contracts/multi_amb_erc20_to_erc677/HomeMultiAMBErc20ToErc677.sol

```diff
@@ -165,7 +165,6 @@ contract HomeMultiAMBErc20ToErc677 is
         // This lock is to prevent calling passMessage twice if a ERC677 token is used.
         // When transferFrom is called, after the transfer, the ERC677 token will call onTokenTransfer from this contract
         // which will call passMessage.
-        require(!lock());
```

Comment:
I agree that the check whether the operation is locked in `bridgeSpecificActionsOnTokenTransfer` is redundant. But it seems that it safe to double check that `relayTokens` is not called by some token contract one more time as part of the `transferFrom` invocation. What do you think?

Reply:
Ok, even if it seems strange to me that the token contract itself will try to relay different tokens as part of the `transferFrom` operation, this check still helps to verify that the lock was not set somewhere earlier in the transaction.
So let's leave it as it was, I am ok with that.